tests/dummy/app/templates/components/sortable.hbs

```diff
@@ -0,0 +1,204 @@
+<h2>Sortable</h2>
+<hr />
```

Comment:
The indentation and style on this page is not the same as in all the others.. Please style it exactly like the others.